src/Disks/ObjectStorages/VFSTraits.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <base/defines.h>
+#include <fmt/format.h>
+
+namespace DB
+{
+struct VFSTraits
+{
+    explicit VFSTraits(std::string_view disk_name)
+        : base_node(fmt::format("/vfs_log/{}", disk_name))
```

Comment:
maybe better to make it configurable? i.e. different clusters use same zookeeper and both have disk called 's3vfs'

Reply:
Will be solved using `vfs_disk_id`